js/scripts.js

```diff
@@ -0,0 +1,18 @@
+
+var text ='Papugi kosza trawe, unoszac sie nad ziemia 15 centymetrów, na czerwonych dywanach.'
+var animal = 'Zielone slonie';
+var animalUpperCased = animal.toUpperCase();
+text = text.split('Papugi').join(animalUpperCased);
+var partOfText = text.slice(0,text.length/2);
+console.log(partOfText);
+
+
+// tutaj raz działa i raz nie działa raczej jest to zle. Ogólne to problem polega na zamianie pierwszego słowa bez podania go w split, bo tak to działa
+
+var text ='Papugi koszą trawę, unosząc się nad ziemią 15 centymetrów, na czerwonych dywanach.';
+var animal = 'Zielone słonie';
+var animalUpperCased = animal.toUpperCase();
+var textCharsAfter = text.replace(theString, animalUpperCased);
+theString = text.split(" ");	
```

Comment:
czemu to się dzieje PO 15 linijce (a w 15 linijce tego używasz przecież)?

daj sobie w konsoli `text.split('')` i zobacz czym to jest, podpowiem, że tablicą - do jej pierwszego elementu możesz dostać się poprzez index np jeśli `array` to tablica to `array[0]` to jej pierwszy element